src/StdChains.sol

```diff
@@ -118,18 +121,29 @@ abstract contract StdChains {
         idToAlias[chain.chainId] = chainAlias;
     }
 
+    function _toUpper(string memory str) private pure returns (string memory) {
```

Comment:
Do we have a source we can link to for where this came from originally? (assuming you didn't write it just for this PR, but maybe you did)

Reply:
I wrote it. :)